src/app/shared/constants/flags/spell_dbc_effect_flags.ts

```diff
@@ -0,0 +1,88 @@
+import { Flag } from '@keira-types/general';
```

Comment:
please use dash `-` instead of underscore in the file name `spell_dbc_effect_flags.ts` -> `spell-dbc-effect-flags.ts`